src/test/resources/DatabaseQueueComponentTest-context.xml

```diff
@@ -39,9 +39,13 @@
       <to id="producer_mock_to" uri="mock:mock_endpoint"/>
     </route>
     <route id="consumer_test_route">
-      <from id="consumer_from" uri="sdpqueue:message_queue?dataSource=sdpqDataSource"/>
+      <from id="consumer_from" uri="sdpqueue:message_queue?dataSource=sdpqDataSource&amp;initialDelay=7"/>
       <to id="consumer_mock_to" uri="mock:mock_endpoint"/>
     </route>
+    <route id="delayed_consumer_test_route">
```

Comment:
It looks like both of these routes have a delay -- the first initial, the second between polls.  Can you rename them to something clearer for people coming across them?